### DISCUSSION TITLE
Universal Search

### DISCUSSION THREAD

**Discussion by dlamkins** [blish-hud/Community-Module-Pack#147] [Module Profiles]
![searchexample](https://user-images.githubusercontent.com/1950594/197963395-75907425-5a5c-4c1a-a7d9-6c1da167b9d1.png)

## Summary

Quickly search for in-game landmarks, skills, and traits by name.  Once found, you can select a result to immediately copy the chat code to your clipboard.

**Comment by kazerniel** [comment]
Hello, is this abandoned? It doesn't seem to support Janthir Wilds landmarks.
